Src/SwqlStudio/MainForm.resx

```diff
@@ -127,7 +127,7 @@
   <data name="menuEditCut.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
     <value>
         iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAO
-        wwAADsMBx2+oZAAAARNJREFUOE+Vkj2OgzAQRn2EPcIeIWegR9qWU3CFvUEOQ09PlS3olyLZhiKJ+Csd
+        wQAADsEBuJFr7QAAARNJREFUOE+Vkj2OgzAQRn2EPcIeIWegR9qWU3CFvUEOQ09PlS3olyLZhiKJ+Csd

```

Comment:
Why the change to the image?

Reply:
There were loads of scaling changes done by the designer when I added the new export options. I thought I undid all of them. I must have missed that one. I'll take it out